src/gdata/docs/service.py

```diff
@@ -331,7 +331,7 @@ def Download(self, entry_or_id_or_url, file_path, export_format=None,
         url = entry_or_id_or_url
 
     if export_format is not None:
-      if url.find('/Export?') == -1:
+      if (url.find('/Export?') == -1) and (url.find('/export?') == -1):
```

Comment:
Can we refactor this to something like
{{{

"/export?" not in url.lower()

}}}